### ISSUE TITLE
Hell banned users

### ISSUE THREAD

**Issue by heroic** [SupportBee/hackerstreet.in#14]
Should we add hellbanned users?

**Comment by bopanna** [comment]
@heroic Actually i have implemented the kill,blast,nuke features given to  admin in hackernews...you can see that the in user table there is an admin string...if you set this variable to "admin" for a user then that user will get to see these options

**Comment by bopanna** [comment]
@heroic  also there is another boolean called ignore in user table...whick becomes "true" if you  blast or nuke a user...thereby making his story dead and unseen to other users...kill can be used to make an individual story as "dead". (Same principle as HSI)

**Comment by heroic** [comment]
Yeah, but what I meant was, that a repeat offender be permanently marked as
hell banned, so that even if he posts, it's automatically nuked.

**Comment by bopanna** [comment]
oh ok...got you, go ahead

**Comment by unamashana** [comment]
@heroic This is a good idea. Can you send us a pull request? Do you know how to attach it to this issue? Otherwise I can do a quick writeup

**Comment by heroic** [comment]
I'll work on it and make it a pull req.
